src/main.rs

```diff
@@ -38,6 +41,47 @@ fn bind_socket(addr: SocketAddr) -> UdpSocket {
     socket
 }
 
+/// Save DHT keys to a binary file.
+fn save_keys(keys_file: String, pk: PublicKey, sk: SecretKey) {
+    #[cfg(not(unix))]
+    let mut file = File::create(keys_file).expect("Failed to create the keys file");
```

Comment:
You should point the name of the file

Reply:
It's logged before with INFO level, like this:
```
 INFO 2018-06-29T17:06:07Z: tox_node: Generating new DHT keys and storing them to /my_file
thread 'main' panicked at 'Failed to create the keys file: Os { code: 13, kind: PermissionDenied, message: "Permission denied" }', libcore/result.rs:945:5
```
So is it really necessary? Because for that I'll have to change `expect` to `panic!` macro with `if let`.

Reply:
Well, no need. Thanks